src/components/_glossary.scss

```diff
@@ -0,0 +1,44 @@
+/// Glossary - used on glossary page
+/// Overrides
+/// See https://www.nice.org.uk/glossary for example
+// sass-lint:disable
+
+.nav-glossary {
+  @include nds-clearfix;
+  display: flex;
+  flex-wrap: wrap;
+  & > li {
+    margin: 0 1% 2%;
```

Comment:
If we're using flexbox then can we use space between rather than using a margin?

Reply:
`space-between` leaves the bottom two items evenly spread, I think for our purposes, fixing the width and wrapping them left is the best option